tests/test_scheduler.rs

```diff
@@ -157,7 +158,7 @@ fn create_config(
                     PlanConfig {
                         id: "managed_robot_zip".into(),
                         source: Source::Managed {
-                            zip_file: "C:\\zips\\restry_rcc_defN.zip".into(),
```

Comment:
It probably makes sense to keep the type of the tar file in the file name, here defN. In the future we can then add other compression methods, and passwords, to know what does and doesn't work.